unity-client/ProjectSettings/ProjectSettings.asset

```diff
@@ -577,7 +577,7 @@ PlayerSettings:
   webGLAnalyzeBuildSize: 0
   webGLUseEmbeddedResources: 0
   webGLCompressionFormat: 0
-  webGLWasmArithmeticExceptions: 0
+  webGLWasmArithmeticExceptions: 1
```

Comment:
are you sure the arithmetic exceptions are being toggled off here? since I see it changes from 0 to 1

Reply:
0 == Throw
1 == Ignore

Not very intuitive but it works.
![image](https://user-images.githubusercontent.com/6875814/114917483-619e0f00-9dfc-11eb-8820-354ebb760b4f.png)